.gitignore

```diff
@@ -13,3 +13,4 @@
 
 # Dependency directories (remove the comment below to include it)
 # vendor/
+.idea/
```

Comment:
Previously we were told don't add this here because people should git ignore their own IDEs from their home directory gitignore. But I am happy to add it 👍

Reply:
thank you! completely agree